src/software/thunderscope/robot_communication.py

```diff
@@ -251,7 +265,7 @@ def __enter__(self):
 
         return self
 
-    def __exit__(self):
+    def __exit__(self, type, value, traceback):
```

Comment:
does thunderscope shutdown gracefully when you hit x now and not freak out with wierd stack message?

Reply:
The process does exit now without any stack trace being printed, though there might be some hidden errors